contracts/src/events.cairo

```diff
@@ -23,3 +23,9 @@ struct EndTurn {
     game_id: felt252,
     turn: felt252,
 }
+
+#[derive(Drop, Serde)]
+struct EndGame {
+    game_id: felt252,
+    round_number: u8,
```

Comment:
why do we want the round number here ? 
This should be added in the end turn

Reply:
> The events log the round number
I understood this requirement as emit the current round number at the moment the game was end. Isn't 1 round contains multiple turns and 1 game contains multiple rounds?

Reply:
It shouldn't be in the end game event at the end of the game. We don't care about the round number when the game is finished. There should probably be a EntRound event which would hold the round number and would be emitted in the `end_turn` system if a round is ending